src/bcm2835.cpp

```diff
@@ -175,6 +176,16 @@ uint32_t* bcm2835_regbase(uint8_t regbase)
     return (uint32_t *)MAP_FAILED;
 }
 
+struct PDESettings
+{
+	uint32_t longDuration;
+	uint32_t shortDuration;
+	uint32_t separatorDuration;
+	uint32_t separator;
+};
+
+std::map<uint32_t, PDESettings> bcm2835_pin_settings_map = {};
```

Comment:
This dependency on map makes it C incompatible... Not sure how to sanely get around it without dropping in a whole map replacement or consuming way more memory than necessary.